cirq-superstaq/cirq_superstaq/compiler_output.py

```diff
@@ -154,11 +167,13 @@ def read_json_aqt(  # pylint: disable=missing-param-doc
     """Reads out returned JSON from SuperstaQ API's AQT compilation endpoint.
 
     Args:
-        json_dict: a JSON dictionary matching the format returned by /aqt_compile endpoint
-        circuits_is_list: bool flag that controls whether the returned object has a .circuits
-            attribute (if True) or a .circuit attribute (False)
+        json_dict:  JSON dictionary matching the format returned by aqt_compile endpoint.
```

Comment:
nit: stray whitespace
```suggestion
        json_dict: JSON dictionary matching the format returned by aqt_compile endpoint.
```